handlers/user-qr.js

```diff
@@ -48,10 +51,11 @@ module.exports = {
             timeout: 8000,
          }).catch((err) => {
             req.log("Error posting registration token to MCC", err);
+            throw err;
```

Comment:
I'm not sure what will happen with this `throw err;`

Since we are not `await ABRelay.post()` it is possible that the next `cb(err, qrCode)` will return before the `.catch()` happens.  

in that case, the `throw err;` should be caught in the `} catch(e) {  cb(e); }` below and the callback will be called 2x.

That second call is unpredictable.

Reply:
Nice catch :)
I'm adding the `await` now.

With the old way, I think my intention was to display the QR code to the user as quickly as possible. Sending the request to the relay might take maybe a second, but it would be fine to display the QR code first, because by the time the user takes out their phone and scans the code, the relay should have gotten the data. And the chances of something going wrong with the relay were slim.

Now I'm thinking that catching a potential problem with the relay is more important. The user is probably okay with waiting a second longer for the QR code to appear, but their experience would be bad if they scanned a QR code that doesn't work.